src/main/java/io/micronaut/build/MicronautGradleEnterprisePlugin.java

```diff
@@ -39,6 +40,7 @@ public void apply(Settings settings) {
         PluginManager pluginManager = settings.getPluginManager();
         pluginManager.apply(MicronautBuildSettingsPlugin.class);
         pluginManager.apply(GradleEnterprisePlugin.class);
+        pluginManager.apply(TestDistributionPlugin.class);
```

Comment:
What is the effect of applying this plugin if predictive test selection is disabled?

Reply:
Nothing, as far as I understand, since we wouldn't configure test selection nor distribution.